### ISSUE TITLE
Create git tags for releases

### ISSUE THREAD

**Issue by MarijnS95** [jean-airoldie/zeromq-src-rs#35]
From the log it is a tad hard to follow what commits have been included in a certain release.

It is probably not worth tagging older releases, but in any case the git hash can be reconstructed from the `.cargo_vcs_info.json` file in the source package, i.e.: https://docs.rs/crate/zeromq-src/0.2.6+4.3.4/source/.cargo_vcs_info.json

**Comment by jean-airoldie** [comment]
Added [tag for latest release](https://github.com/jean-airoldie/zeromq-src-rs/tree/v0.3.0+4.3.5). If you need a specific older tag, just tell me.

**Comment by MarijnS95** [comment]
Cool! Having tags for at least the latest 2-3 releases (`0.2.3-0.2.6`) might be nice too, as folks are still using those.

**Comment by MarijnS95** [comment]
https://crates.io/crates/zeromq-src even "other" releases are still used.

**Comment by jean-airoldie** [comment]
Added tags for 0.2.4 to 0.2.6, which i validated against the crates.io hash. I skipped 0.2.3 because the version is yanked, due to a problem with cloning submodules, if i recall correctly.

**Comment by MarijnS95** [comment]
@jean-airoldie thanks! I don't think it would hurt having tags for _yanked_ releases though, this still makes it clear at what point in the tree such a "broken and yanked" release was created.
